src/main/resources/create_custom_fields_table.sql

```diff
@@ -0,0 +1 @@
+CREATE TABLE IF NOT EXISTS custom_fields(refId VARCHAR(50) PRIMARY KEY , jsonb JSONB NOT NULL)
```

Comment:
Is there a reason why we are not naming this "id" and using "refId" - I understand that an app(users) references this table and record but not sure if having "refId" as the primary key column name is consistent with FOLIO's way

Reply:
I think you are right, I ran into some problems with naming it refId earlier, so I will rename it to id

Reply:
👍